test/cqlpy/test_vector_index.py

```diff
@@ -103,4 +103,10 @@ def test_describe_custom_index(cql, test_keyspace):
         b_desc = cql.execute(f"DESC INDEX {test_keyspace}.custom1").one().create_statement
 
         assert f"CREATE CUSTOM INDEX custom ON {table}{maybe_space}(v1) USING '{custom_class}'" in a_desc
-        assert f"CREATE CUSTOM INDEX custom1 ON {table}{maybe_space}(v2) USING '{custom_class}'" in b_desc
\ No newline at end of file
+        assert f"CREATE CUSTOM INDEX custom1 ON {table}{maybe_space}(v2) USING '{custom_class}'" in b_desc
+
+        # Try to recreate the indexes with the describe output
+        cql.execute(f"DROP INDEX {test_keyspace}.custom")
+        cql.execute(f"DROP INDEX {test_keyspace}.custom1")
+        cql.execute(a_desc)
```

Comment:
If I understand correctly, this just verifies that a_desc is not a syntax error, but nothing else about it. You already verified a few lines up that a_desc contains exactly what you wanted to contain, so what's the point of these extra drop and execute?